src/events/mouse.js

```diff
@@ -929,4 +969,44 @@ p5.prototype._onwheel = function(e) {
   }
 };
 
+/**
+ * <p>The function <a href="#/p5/requestPointerLock">requestPointerLock()</a>
+ * locks the pointer to its current position and makes it invisible.
+ * Use deltaX and deltaX to get the difference the mouse was moved since
```

Comment:
You can link deltaX and deltaY to their reference pages. See this code snippet as an example of linking: https://github.com/processing/p5.js/blob/master/src/image/loading_displaying.js#L22

Reply:
https://github.com/processing/p5.js/pull/3804/commits/2fcfa818300f93b63b3363efc8670254fe2e0fc6 added links